torch_xla/experimental/pjrt.py

```diff
@@ -211,11 +211,9 @@ def _merge_replica_results(
 
 
 @requires_pjrt
-def _run_thread_per_device(local_rank: int,
-                           local_world_size: int,
-                           fn: Callable[[], R],
-                           initializer_fn: Callable[[int, int], None],
-                           master_port: int = 12355) -> Dict[int, R]:
```

Comment:
so we don't need master port anymore?

Reply:
I also did a clean up of unused variables in the functions that I touched. Sorry, should have included that in the PR description.